app/ts/components/Submit.tsx

```diff
@@ -229,18 +232,29 @@ export const Submit = ({
 		}
 	}
 
+	function showSettingsModal() {
+		showSettings.value = true
+	}
+
 	return (
 		<>
 			<h2 className='font-bold text-2xl'>3. Submit</h2>
+			<SettingsModal display={showSettings} appSettings={appSettings} />
 			{missingRequirements.value ? (
 				<p>{missingRequirements.peek()}</p>
 			) : (
-				<div className='flex flex-col w-full gap-6'>
-					<Button onClick={simulateBundle} variant='secondary'>
-						Simulate
-					</Button>
+				<div className='flex flex-col w-full gap-4'>
+					<div>
+						<p><span className='font-bold'>Gas:</span> {utils.formatUnits(appSettings.value.priorityFee.toString(), 'gwei')} gwei + {utils.formatUnits(appSettings.value.priorityFee.toString(), 'gwei')} gwei priority</p>
```

Comment:
```suggestion
						<p><span className='font-bold'>Gas:</span> {utils.formatUnits(appSettings.value.priorityFee.toString(), 'gwei')} nanoeth + {utils.formatUnits(appSettings.value.priorityFee.toString(), 'gwei')} nanoeth priority</p>
```

Reply:
we should use SI units

Reply:
Most common units people use for gas is gwei, nanoeth may be confusing for users if its not common on other apps.